src/electron-forge-init.js

```diff
@@ -17,7 +18,8 @@ const main = async () => {
   program
     .version(require('../package.json').version)
     .arguments('[name]')
-    .option('-l, --lintstyle [style]', 'Linting standard to follow.  Can be "airbnb" or "standard"', 'airbnb')
+    .option('-t, --template [name]', 'Name of the forge template to use')
+    .option('-l, --lintstyle [style]', 'Linting standard to follow.  For the default template it can be "airbnb" or "standard"', 'airbnb')
```

Comment:
@malept Technically this is passed through to custom templates, so they can use it as well